__tests__/data/node-nightly-index.json

```diff
@@ -0,0 +1,35 @@
+[
```

Comment:
Do we have docs for maintainers on where these files come from and when to update them?